spotify.html

```diff
@@ -113,70 +130,53 @@ <h2>Get even more questions answered <u>here.</u></h2>
 
         </div>
 
-        <div class="Line8">
-            <ul id="accordion">
-                <li>
-                    <label for="first">What is an Accordion? <span>&#x3e</span></label>
-                    <input type="radio" name="accordion" id="first" checked>
-
-                    <div>
-                        <p>
-                            Lorem ipsum dolor sit amet consectetur adipisicing elit. Incidunt, cupiditate? Reprehenderit veniam iure ab necessitatibus voluptates nemo numquam, laudantium ratione totam, qui incidunt voluptatibus soluta hic error culpa cum repudiandae?
-                        </p>
-                    </div>
-                </li>
-
 
-                <li>
-                    <label for="second">What is an Accordion? <span>&#x3e</span></label>
-                    <input type="radio" name="accordion" id="second" >
-
-                    <div>
-                        <p>
-                            Lorem ipsum dolor sit amet consectetur adipisicing elit. Incidunt, cupiditate? Reprehenderit veniam iure ab necessitatibus voluptates nemo numquam, laudantium ratione totam, qui incidunt voluptatibus soluta hic error culpa cum repudiandae?
-                        </p>
-                    </div>
-                </li>
-
-
-
-                <li>
-                    <label for="third">What is an Accordion? <span>&#x3e</span></label>
-                    <input type="radio" name="accordion" id="third">
-
-                    <div>
-                        <p>
-                            Lorem ipsum dolor sit amet consectetur adipisicing elit. Incidunt, cupiditate? Reprehenderit veniam iure ab necessitatibus voluptates nemo numquam, laudantium ratione totam, qui incidunt voluptatibus soluta hic error culpa cum repudiandae?
-                        </p>
-                    </div>
-                </li>
-
-
-                <li>
-                    <label for="forth">What is an Accordion? <span>&#x3e</span></label>
-                    <input type="radio" name="accordion" id="forth">
-
-                    <div>
-                        <p>
-                            Lorem ipsum dolor sit amet consectetur adipisicing elit. Incidunt, cupiditate? Reprehenderit veniam iure ab necessitatibus voluptates nemo numquam, laudantium ratione totam, qui incidunt voluptatibus soluta hic error culpa cum repudiandae?
-                        </p>
-                    </div>
-                </li>
-
-
-                <li>
-                    <label for="fifth">What is an Accordion? <span>&#x3e</span></label>
-                    <input type="radio" name="accordion" id="fifth">
+        <div class="line8">
+            <div> <img src="img/Spotify_Logo_RGB_White.png">
+            </div>
+    
+            <div class="text9"> <h2>COMPANY</h2>
+                <h3>About</h3>
```

Comment:
Ideally this should be link tags here instead of h3 ` <a href="/about">About</a>` you might need to wrap the parent class in a list